app/data_model/session_store.py

```diff
@@ -76,11 +76,10 @@ def _load(self):
             self.user_id = self._eq_session.user_id
 
             if self._eq_session.session_data:
-                session_data = self._eq_session.session_data
-                if self._is_session_data_encrypted(session_data):
-                    session_data = StorageEncryption(self.user_id, self.user_ik, self.pepper).decrypt_data(session_data)
+                encrypted_session_data = self._eq_session.session_data
+                encrypted_session_data = StorageEncryption(self.user_id, self.user_ik, self.pepper).decrypt_data(encrypted_session_data)
```

Comment:
After getting encrypted_session_data the next variable should just be session_data. This shows that this step is around decrypting the stored encrypted data. So line 79 you get encrypted_session_data and then on line 80 you use that to create session_data which is then used to create the self.session_data on line 82.